mithril-common/src/messages/epoch_settings.rs

```diff
@@ -0,0 +1,72 @@
+use crate::entities::{Epoch, ProtocolParameters};
+use serde::{Deserialize, Serialize};
+
+/// EpochSettings represents the settings of an epoch
+#[derive(Clone, Debug, PartialEq, Default, Serialize, Deserialize)]
+pub struct EpochSettingsMessage {
+    /// Current Epoch
+    pub epoch: Epoch,
+
+    /// Current Protocol parameters
+    #[serde(rename = "protocol")]
+    pub protocol_parameters: ProtocolParameters,
+
+    /// Next Protocol parameters
+    #[serde(rename = "next_protocol")]
+    pub next_protocol_parameters: ProtocolParameters,
+}
+
+impl EpochSettingsMessage {
+    /// Dummy instance for test purposes.
+    pub fn dummy() -> Self {
+        Self {
+            epoch: Epoch(10),
+            protocol_parameters: ProtocolParameters {
+                k: 5,
+                m: 100,
+                phi_f: 0.65,
+            },
+            next_protocol_parameters: ProtocolParameters {
+                k: 5,
+                m: 100,
+                phi_f: 0.65,
+            },
+        }
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    fn golden_message() -> EpochSettingsMessage {
+        EpochSettingsMessage {
+            epoch: Epoch(10),
+            protocol_parameters: ProtocolParameters {
+                k: 5,
+                m: 100,
+                phi_f: 0.65,
+            },
+            next_protocol_parameters: ProtocolParameters {
+                k: 5,
+                m: 100,
+                phi_f: 0.65,
```

Comment:
Maybe we wnt different values for the next protocol parameters?
```suggestion
                k: 50,
                m: 1000,
                phi_f: 0.65,
```